README.rst

```diff
@@ -15,3 +15,40 @@ The code in this repository is licensed under the Not open source unless
 otherwise noted.
 
 Please see `LICENSE.txt <LICENSE.txt>`_ for details.
+
+Installation and Configuration
```

Comment:
I forgot about one thing - [this part](https://github.com/open-craft/openedx-completion-aggregator/?tab=readme-ov-file#synchronous-vs-asynchronous-calculations) is essential to read before enabling this in a production environment, as installing this package can lead to an outage. We should probably use [a caution highlight](https://github.com/orgs/community/discussions/16925) in this README to make this more visible.

Reply:
Good call. Added it.